scripts/run_tests.sh

```diff
@@ -0,0 +1,27 @@
+apt-get update -y && apt-get install -y wget python3-pip && \
+pushd geopyspark-backend && ./sbt "project geotrellis-backend" assembly && \
+cp geotrellis/target/scala-2.11/geotrellis-backend-assembly-*.jar ../geopyspark/jars && popd && \
+if [ ! -f archives/spark-2.3.2-bin-hadoop2.7.tgz ]; then
+  cd archives
+  wget http://apache.cs.utah.edu/spark/spark-2.3.2/spark-2.3.2-bin-hadoop2.7.tgz
```

Comment:
http://apache.cs.utah.edu/spark/spark-2.3.1/spark-2.3.1-bin-hadoop2.7.tgz is not valid anymore; see http://apache.cs.utah.edu/spark/